workspaces/ui-v2/src/optic-components/hooks/util/useRunOnKeypress.ts

```diff
@@ -0,0 +1,35 @@
+import { KeyboardEvent, useCallback } from 'react';
+
+export const useRunOnKeypress = <T extends (...args: any) => any>(
```

Comment:
This is something we could use for the future if we wanted to better support power users (keyboard shortcuts to accept, etc)

Reply:
This looks cool, but I would prefer if we stuck to standard web form ux, unless that is way more complicated. I do see the value of supporting future shortcuts, but for now I'd focus on just making sure people don't have to unnecessarily switch between keyboard and mouse throughout the ux. Thoughts?